urllib3/connectionpool.py

```diff
@@ -705,8 +706,8 @@ def urlopen(self, method, url, body=None, headers=None, retries=None,
                     response.release_conn()
                     raise
                 return response
-            retries.sleep()
-            log.debug("Forced retry: %s", url)
+            retries.sleep(response)
+            log.debug("Retry: %s", url)
```

Comment:
Because this line runs if `retry.is_retry()` returns true. `retry.is_retry()` may return true even if the stauts code is not in `status_forcelist`.